net/network.go

```diff
@@ -53,3 +103,77 @@ func (network *Network) GetPeerID() peer.ID {
 func (network *Network) GetBandwidthStats() metrics.Stats {
 	return network.Reporter.GetBandwidthTotals()
 }
+
+// Connect connects to peers at the given addresses
+func (network *Network) Connect(ctx context.Context, addrs []string) ([]peer.ID, error) {
+	swrm, ok := network.host.Network().(*swarm.Swarm)
+	if !ok {
+		return nil, fmt.Errorf("peerhost network was not a swarm")
+	}
+
+	pis, err := PeerAddrsToPeerInfos(addrs)
+	if err != nil {
+		return nil, err
+	}
+
+	output := make([]peer.ID, len(pis))
+	for i, pi := range pis {
+		swrm.Backoff().Clear(pi.ID)
+
+		output[i] = pi.ID
+
+		if err := network.host.Connect(ctx, pi); err != nil {
+			return nil, errors.Wrapf(err, "peer: %s", output[i].Pretty())
+		}
+	}
+
+	return output, nil
+}
+
+// Peers lists peers currently available on the network
+func (network *Network) Peers(ctx context.Context, verbose, latency, streams bool) (*SwarmConnInfos, error) {
+	if network.host == nil {
+		return nil, errors.New("node must be online")
+	}
+
+	conns := network.host.Network().Conns()
+
+	var out SwarmConnInfos
+	for _, c := range conns {
+		pid := c.RemotePeer()
+		addr := c.RemoteMultiaddr()
+
+		ci := SwarmConnInfo{
+			Addr: addr.String(),
+			Peer: pid.Pretty(),
+		}
+
+		/* FIXME(steb)
```

Comment:
can we turn this into an issue or just delete it?

Reply:
Deleted.